### PR TITLE
Fix template and refactory code

### PR DESCRIPTION
**Pull request by at-trungtranqn**
- Fix template html admin place
- Refactory code admin place

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -80,54 +80,73 @@
...
-.btn-pencil{
-    font-size: 18px;
+.btn-option .btn-edit {
+    margin-top: 7px;
+    margin-left: 21px;
 }
 
-.col-no {
-    width: 5%;
+.btn-option a  {
```
Comment (ghost) [inline public/css/main.css]: Bỏ cái này đi. Dùng class `pull-left` trực tiếp vào nó.

File: public/css/main.css [hunk 2]
```diff
@@ -80,54 +80,73 @@
...
+    margin-top: 10px;
+}
+.place-descript{
+    max-height: 300px;
+    max-width: 713px;
+}
+.btn-custom {
+    margin-right: 10px;
+}
+.btn-submit{
```
Comment (ghost) [inline public/css/main.css]: remove this, boostrap has class `pull-right` is float : right, you can using it.

**Review by ghost — COMMENTED** [review]
Check giúp vs nha!

File: resources/views/backend/places/create.blade.php [hunk 3]
```diff
@@ -40,19 +40,28 @@
...
               <div class="box-footer">
-                <button type="reset" class="btn btn-warning">{{ __("Reset") }}</button>
-                <button type="submit" class="btn btn-primary">{{ __("Submit") }}</button>
+                <a class="btn btn-default btn-custom" href="{{route('place.index')}}">
+                  {{ __('Back') }}
+                </a>
+                <button type="reset" class="btn btn-warning btn-custom">
+                  {{ __('Reset') }}
+                </button>
+                <button type="submit" class="btn btn-primary btn-custom btn-submit">
```
Comment (ghost) [inline resources/views/backend/places/create.blade.php]: replace `btn-submit` by `pull-right` if you want float : right

File: public/js/main.js [hunk 4]
```diff
@@ -9,5 +9,10 @@ $(document).ready(function(){
         return false;
       }
     });
+
+    $('#preview-image').change( function(event) {
+        var imgpath = URL.createObjectURL(event.target.files[0]);
```
Comment (ghost) [inline public/js/main.js]: please check if not choose image : 
**step 1** : you click and choose a image.
**step 2** : you click again and not choose image. please set image default again.

Comment (at-trungtranqn) [reply public/js/main.js]: Why need set image default again? 
I'm has required image input not null, if you not choose image on second click, you can see last image after first click.Sure, you can't to leave blank input image with image default and submit.
Explain for me again, please !

Comment (at-trungtranqn) [reply public/js/main.js]: i was fixed, review again help me, please !

**Review by ghost — COMMENTED** [review]
LGTM.

**Comment by ghost** [comment]
images

**Pull request merged**